streams/src/main/resources/common/message/SubscriptionInfo.json

```diff
@@ -0,0 +1,71 @@
+// Licensed to the Apache Software Foundation (ASF) under one or more
+// contributor license agreements.  See the NOTICE file distributed with
+// this work for additional information regarding copyright ownership.
+// The ASF licenses this file to You under the Apache License, Version 2.0
+// (the "License"); you may not use this file except in compliance with
+// the License.  You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+{
+  "name": "SubscriptionInfo",
+  "validVersions": "1-5",
+  "fields": [
+    {
+      "name": "version",
+      "versions": "1+",
+      "type": "int32"
+    },
+    {
+      "name": "latestSupportedVersion",
+      "versions": "3+",
+      "default": "-1",
+      "type": "int32"
+    },
+    {
+      "name": "processId",
```

Comment:
Should we also define a default value here?

Reply:
We don't need to. The UUID type is non-nullable (at the request of Colin when I added UUID support), so any time it's missing, a default value is filled in. The default is a "zero uuid" aka a uuid consisting only of zeros.